packages/components/src/toolbar/toolbar/index.tsx

```diff
@@ -19,23 +19,30 @@ import type { ToolbarProps } from './types';
 import type { WordPressComponentProps } from '../../context';
 import { ContextSystemProvider } from '../../context';
 
-const CONTEXT_SYSTEM_VALUE = {
-	DropdownMenu: {
-		variant: 'toolbar',
-	},
-	Dropdown: {
-		variant: 'toolbar',
-	},
-};
-
 function UnforwardedToolbar(
 	{
 		className,
 		label,
+		variant,
 		...props
 	}: WordPressComponentProps< ToolbarProps, 'div', false >,
 	ref: ForwardedRef< any >
 ) {
+	const CONTEXT_SYSTEM_VALUE = useMemo( () => {
```

Comment:
I'd encourage changing `CONTEXT_SYSTEM_VALUE` to `contextSystemValue` since this is no longer a module constant.

Reply:
I'm also unfamiliar with the term 'module constants' so thanks for suggesting! My assumption on this change is now there is more than one possible return value vs. before. Is that correct, or is it based on something else?

Reply:
This simply a naming convention: constant values variables are usually UPPERCASE_LIKE_THIS. That was the case for `CONTEXT_SYSTEM_VALUE` before this PR.

But since it's been moved inside the react component's function, it's no longer a constant. Marin is suggesting that we change the casing (if that's even a word) to a more traditional camelCase

Reply:
Thanks for elaborating, @ciampo, that's exactly what I meant.